src/main/java/org/folio/innreach/domain/service/impl/BatchDomainEventProcessor.java

```diff
@@ -47,11 +55,13 @@ private <T> void processTenantEvents(List<DomainEvent<T>> events, Consumer<Domai
     for (var event : events) {
       log.info("Processing event {}", event);
       try {
-        retryTemplate.execute(ctx -> {
           recordProcessor.accept(event);
-          return null;
-        });
-      } catch (Exception e) {
+      }
+      catch (ServiceSuspendedException | FeignException | InnReachConnectionException e) {
+        log.info("exception thrown from process");
```

Comment:
Same as above. Let's log the exception so we can know which one of the three happened.